model/src/main/java/io/github/droidkaigi/confsched2018/model/Session.kt

```diff
@@ -37,4 +37,7 @@ sealed class Session(
 
     val isFinished: Boolean
         get() = System.currentTimeMillis() > endTime.time
+
+    val isOnGoing: Boolean
+        get() = Date().after(startTime) && Date().before(endTime)
```

Comment:
Can you use a single instance for this? 🙇 💦 
For performance and safety.

```
System. currentTimeMillis().let { time -> startTime.time < time && time < endTime.time }
```

or

```
Date().let { date -> date.after(startTime) && date.before(endTime) }
```

Reply:
I guess `System.currentTimeMillis() in startTime.time..endTime.time` is smart and can satisfy the concern @takahirom mentioned. 🤔

Reply:
it's the best. 🥇

Reply:
Thank you for your advices. I fixed it following jmatsu's one.